guides/Wrtiting scannable titles and headings.md

```diff
@@ -0,0 +1,62 @@
+# Writing document titles and headings optimized for search
+
+In Expo documentation, document _titles_ and section _headings_ provide structure and reference points to help developers search and scan any document. Part of our job is to help them reach the point they need to solve their problem.
+
+Following suggestions and tips from this guide will help you to write document titles and section headings that are easy to search and make a document scannable.
+
+---
+
+Let's start with an example where a heading is unaligned to its document's content:
+
+[<img src="./assets/writing-guides/01-search-doc-example.png" width="540" />](./assets/writing-guides/01-search-doc-example.png)
+
+Based on the search term above, the correct page appears as the last item. The search term contains all keywords that are common with the way we talk about running an EAS Build on a local computer.
+
+Using the context of a keyword (commonly used term), it improves the search ranking of the document when the same search term is used:
```

Comment:
```suggestion
Using keywords in the heading improves the search ranking of the document when the same search term is used:
```

Reply:
not exactly; I meant to say here that some sort of context that represents the keyword. A keyword in every case will not be an ideal fit. I'll try rephrase it.